pkg/registration/spoke/options.go

```diff
@@ -70,8 +92,15 @@ func (o *SpokeAgentOptions) AddFlags(fs *pflag.FlagSet) {
 
 // Validate verifies the inputs.
 func (o *SpokeAgentOptions) Validate() error {
-	if o.BootstrapKubeconfig == "" {
-		return errors.New("bootstrap-kubeconfig is required")
+	if features.SpokeMutableFeatureGate.Enabled(ocmfeature.MultipleHubs) {
+		// expect BootstrapKubeconfig is empty and BootstrapKubeconfigs has at least 2 items
+		if len(o.BootstrapKubeconfigs) < 2 {
+			return errors.New("expect at least 2 bootstrap kubeconfigs")
+		}
+	} else if !features.SpokeMutableFeatureGate.Enabled(ocmfeature.MultipleHubs) {
+		if o.BootstrapKubeconfig == "" {
```

Comment:
I do not think we need this check. if it is empty, we will just check the kubeconfig in the default location.

Reply:
Does "this check" means the check of the featureGates?

Reply:
if o.BootstrapKubeconfig == ""